x-pack/plugins/security_solution/public/overview/components/detection_response/alerts_by_status/alerts_by_status.tsx

```diff
@@ -151,6 +160,54 @@ export const AlertsByStatus = ({
     []
   );
 
+  const onDonutPartitionClick = useCallback(
+    (level: string, status: Status) => {
+      if (level && status) {
+        navigateToAlerts([
+          {
+            title: 'Severity',
```

Comment:
Not sure if these titles should be translated strings?

Reply:
changed 👌